lib/image-diff.js

```diff
@@ -81,8 +81,8 @@ ImageDiff.createDiff = function (options, cb) {
     }
 
     // Callback with pass/fail
-    var totalDifference = resultInfo[1];
-    return cb(null, totalDifference === '0');
+    var totalDifference = resultInfo[2]; // this is the percentage difference
+    return cb(null, parseFloat(totalDifference) <= options.threshold, options.diffPath);
```

Comment:
And as mentioned in the originally proposed changes, we need to handle when this parses to `NaN`:

https://github.com/uber/image-diff/issues/14#issuecomment-86199714

```js
if (isNaN(percentDifference)) {
  return cb(new Error('Attempted to parse percentage difference from `image-diff\'s stderr` but got `NaN` from "' + resultInfo[2] + '"'));
}
```